packages/components/button/src/Button.tsx

```diff
@@ -49,15 +49,21 @@ export const Button = React.forwardRef<HTMLButtonElement, ButtonProps>(
     ref
   ) => {
     const Component = asChild ? Slot : 'button'
-    const isDisabled = !!disabled || isLoading
+
+    const shouldNotInteract = !!disabled || isLoading
 
     /**
      * When using `asChild` (polymorphic) it's possible that the button becomes another HTML element.
      * Depending on its tag, it could break the `disabled` html attribute preventing the clicks on a disabled button.
      */
     const handleClick = (event: MouseEvent<HTMLButtonElement>) => {
```

Comment:
Should we also handle other events, such as `onKeyDown`, `onKeyPress`etc...🙃🥲😢😭

Reply:
Not sure to understand your remark... Could you precise your point plz?

Reply:
Sure,
```tsx
<Button isLoading onKeyDown={() => fetch('someUrl')}>Default button</Button>
```

The `onKeyDown` handler will not be prevented

Reply:
:crying_cat_face:

Reply:
Code updated to take care of all (?) mouse and keyboar events that should be blocked while `isLoading`. Happy to discuss :+1: